gui/src/components/JobRuns/StatusCard.js

```diff
@@ -3,28 +3,42 @@ import { withStyles } from '@material-ui/core/styles'
 import PaddedCard from 'components/PaddedCard'
 import StatusIcon from 'components/JobRuns/StatusIcon'
 import Typography from '@material-ui/core/Typography'
+import classNames from 'classnames'
 
 const styles = theme => ({
```

Comment:
Why the inline styles here?

Reply:
These aren't inline styles. It's [JSS](https://material-ui.com/customization/css-in-js/).

During the build phase optimized classes with unique names get created and extracted into an external stylesheet.

Reply:
JSS looks like inline styles to me, in that it's styles alongside your view code, rather than styles bundled together along with other styles.

Nevertheless I don't have the background to evaluate the JSS strategy yet, so Imma leave you on your way with this one ;)

Reply:
It does look similar. I think of inline styles as setting the styles directly on the DOM element e.g.

```
<div style="background: 'red'">Hello</div>
```

For background about why???

> CSS uses a global namespace for CSS Selectors that can easily result in style conflicts throughout your application when building an application using modern web components. You can avoid this problem by nesting CSS selectors or use a styling convention like BEM but this becomes complicated quickly and won’t scale.
> 
> CSS-in-JS avoids these problems entirely by generating unique class names when styles are converted to CSS. This allows you to think about styles on a component level with out worrying about styles defined elsewhere.

- https://cssinjs.org/from-sass-to-cssinjs